param/parameterized.py

```diff
@@ -31,17 +31,27 @@
 VERBOSE = INFO - 1
 logging.addLevelName(VERBOSE, "VERBOSE")
 
-# Logger instance to use for param; if "logger" is set to None, the root logger
-# will be used.
+# Get the appropriate logging.Logger instance. If `logger` is None, a
+# logger named `"param"` will be instantiated. If `name` is set, a descendant
+# logger with the name ``"param.<name>"`` is returned (or
+# ``logger.name + ".<name>"``)
 logger = None
-def get_logger():
+def get_logger(name=None):
     if logger is None:
-        # If it was not configured before, do default initialization
-        if not logging.getLogger().handlers:
-            logging.basicConfig(level=INFO)
```

Comment:
Should we keep this?

Reply:
Now that we're making our own logger, I think we should explicitly set the default, yes.